src/EnergyPlus/HVACHXAssistedCoolingCoil.cc

```diff
@@ -1575,9 +1364,6 @@ namespace HVACHXAssistedCoolingCoil {
         // incorrect coil type or name is given, ErrorsFound is returned as true and node number is returned
         // as zero.
 
-        // Using/Aliasing
-        auto &GetWaterCoilWaterInletNode(WaterCoils::GetCoilWaterInletNode);
```

Comment:
[src/EnergyPlus/HVACHXAssistedCoolingCoil.cc:1579]:(style),[constVariable],Variable 'GetWaterCoilWaterInletNode' can be declared as reference to const